codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/localservice/extensions/DafnyPythonLocalServiceStructureGenerator.java

```diff
@@ -411,6 +435,94 @@ protected void writeFromDict(boolean isError) {
     writer.write("");
   }
 
+  /**
+   * Override Smithy-Python writeAsDict to handle shapes with {@link ReferenceTrait}s.
+   * Most of this is lifted directly from Smithy-Python; the changed components are
+   * highlighted.
```

Comment:
The whole thing is highlighted if you meant that the diff would be highlighted (In that case, it's better to remove the highlighted word).

Reply:
Oh no, by "highlighted" I meant that I was calling out specific areas.
But it is unclear. Let me clarify